pkg/auth/oauth/registry/tokentimeouts.go

```diff
@@ -23,11 +22,12 @@ import (
 var ErrTimedout = errors.New("Token timed out")
 
 type tokenData struct {
-	name     string
-	client   string
-	seen     time.Time
-	creation time.Time
-	timeout  int32
+	token *oauth.OAuthAccessToken
```

Comment:
You said you didnt't want to create a cache of tokens, but this is basically a cache, individual fields were explicitly save to avoid carrying the whole token around

Reply:
Sorry I was really unclear what I meant when I said "do not create a cache" - I meant do not create a shared informer cache of _all_ tokens.  By wiring up the OAuth client shared informer, the server actually now has a cache of all tokens as well (nothing is getting data out of it, but the server is keeping it up to date regardless).  This is what I mean by "remove the generated token bits" - we need to kill that generated code by removing the comments that cause its creation.

Keeping the whole token around is fine with me, and we need most of its fields to making the logs useful.  Speaking of logs, I was thinking about adding the SHA256 hash of the token name into the log as well (that way the log does not leak the token value, but can still be tracked to a given token if needed).  WDYT?